library/src/main/avro/pfbSchema.avdl

```diff
@@ -0,0 +1,55 @@
+@namespace("bio.terra.pfb")
+/**
+ *  Built based on example provided by the pyPFB library
+ https://github.com/uc-cdis/pypfb/blob/master/docs/index.md#example
+ For PFB files, the only entity which is MUST be included in the PFB file is Metadata: this
```

Comment:
Xtreme nit: "which is MUST" has an extra "is"!  Only update if you're already making other changes!

Reply:
Thanks! I've updated it!